ipsets/ipsets.go

```diff
@@ -626,7 +630,19 @@ func (s *IPSets) tryResync() (numProblems int, err error) {
 // 'ipset restore' session in order to minimise process forking overhead.  Note: unlike
 // 'iptables-restore', 'ipset restore' is not atomic, updates are applied individually.
 func (s *IPSets) tryUpdates() error {
```

Comment:
I think it's worth deleting IP sets that are no longer needed too; otherwise when we debug from `ipset list` we'll see stale data and that'll be confusing.